modules/RTC/TraceablePeerConnection.js

```diff
@@ -1601,6 +1601,18 @@ TraceablePeerConnection.prototype._assertTrackBelongs = function(
  * video in the local SDP.
  */
 TraceablePeerConnection.prototype.getConfiguredVideoCodec = function() {
+    const localVideoTrack = this.getLocalVideoTracks()[0];
+
+    if (this._usesCodecSelectionAPI && localVideoTrack) {
+        const rtpSender = this.findSenderForTrack(localVideoTrack.getTrack());
+
+        if (rtpSender) {
+            const { codecs } = rtpSender.getParameters();
```

Comment:
This API exists before the new codec selection one. Shouldn't we be using it without the guard?

Reply:
`findSenderForTrack` still checks if `getSenders` on the peerconnection exists and only then returns the `RTCRtpSender` . Now that getSenders is implemented everywhere, I can go and update `findSenderForTrack` and remove the guard from everywhere else as well.